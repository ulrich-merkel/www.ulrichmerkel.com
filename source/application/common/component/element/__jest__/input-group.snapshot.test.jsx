/* global jest, describe, expect */
/* eslint-disable react/prefer-es6-class, react/prefer-stateless-function, func-names, react/prop-types, import/no-extraneous-dependencies */
import React from 'react';
import renderer from 'react-test-renderer';
import ComponentToBeTested from './../input-group';

describe('component/element/input-group', function () {
    describe('Snapshot', function () {
        it('should render correctly', function () {
            var tree = renderer.create(
                <ComponentToBeTested
                    id='foo'
                    name='bar'
                    className='foobar'
                    label='test'
                    type='submit'
                    value='2'
                    isValid
                    isPristine
                >
                    Hello
                </ComponentToBeTested>
            ).toJSON();
            expect(tree).toMatchSnapshot();
        });
    });
});
