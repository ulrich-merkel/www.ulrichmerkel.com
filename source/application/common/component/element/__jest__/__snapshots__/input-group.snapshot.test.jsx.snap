exports[`component/element/input-group Snapshot should render correctly 1`] = `
<div
  className="foobar">
  <input
    aria-required={true}
    className="m-form__control--text is-pristine"
    id="foo"
    name="bar"
    onBlur={[Function]}
    onChange={[Function]}
    placeholder=""
    required={true}
    type="submit"
    value="2" />
  <label
    className="m-form__label"
    htmlFor="foo">
    test
    <i
      aria-hidden="true"
      className="c-font-icon--sad m-form__label__icon--error" />
    <i
      aria-hidden="true"
      className="c-font-icon--smile m-form__label__icon--success" />
  </label>
</div>
`;
